[2022-02-21 16:53:21,442] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: test_path.get_path scheduled__2022-01-29T00:00:00+00:00 [queued]>
[2022-02-21 16:53:21,457] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: test_path.get_path scheduled__2022-01-29T00:00:00+00:00 [queued]>
[2022-02-21 16:53:21,457] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-02-21 16:53:21,457] {taskinstance.py:1239} INFO - Starting attempt 1 of 1
[2022-02-21 16:53:21,457] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-02-21 16:53:21,477] {taskinstance.py:1259} INFO - Executing <Task(PythonOperator): get_path> on 2022-01-29 00:00:00+00:00
[2022-02-21 16:53:21,484] {standard_task_runner.py:52} INFO - Started process 116 to run task
[2022-02-21 16:53:21,493] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'test_path', 'get_path', 'scheduled__2022-01-29T00:00:00+00:00', '--job-id', '32', '--raw', '--subdir', 'DAGS_FOLDER/test.py', '--cfg-path', '/tmp/tmpf6t6neq0', '--error-file', '/tmp/tmp48qhsefi']
[2022-02-21 16:53:21,493] {standard_task_runner.py:77} INFO - Job 32: Subtask get_path
[2022-02-21 16:53:21,546] {logging_mixin.py:109} INFO - Running <TaskInstance: test_path.get_path scheduled__2022-01-29T00:00:00+00:00 [running]> on host 7c24e16e6467
[2022-02-21 16:53:21,610] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=test_path
AIRFLOW_CTX_TASK_ID=get_path
AIRFLOW_CTX_EXECUTION_DATE=2022-01-29T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-01-29T00:00:00+00:00
[2022-02-21 16:53:21,610] {logging_mixin.py:109} INFO - /opt/***
[2022-02-21 16:53:21,610] {python.py:175} INFO - Done. Returned value was: None
[2022-02-21 16:53:21,622] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=test_path, task_id=get_path, execution_date=20220129T000000, start_date=20220221T165321, end_date=20220221T165321
[2022-02-21 16:53:21,662] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-02-21 16:53:21,692] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-02-21 17:02:04,008] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: test_path.get_path scheduled__2022-01-29T00:00:00+00:00 [queued]>
[2022-02-21 17:02:04,018] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: test_path.get_path scheduled__2022-01-29T00:00:00+00:00 [queued]>
[2022-02-21 17:02:04,018] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-02-21 17:02:04,018] {taskinstance.py:1239} INFO - Starting attempt 1 of 1
[2022-02-21 17:02:04,018] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-02-21 17:02:04,028] {taskinstance.py:1259} INFO - Executing <Task(PythonOperator): get_path> on 2022-01-29 00:00:00+00:00
[2022-02-21 17:02:04,032] {standard_task_runner.py:52} INFO - Started process 113 to run task
[2022-02-21 17:02:04,035] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'test_path', 'get_path', 'scheduled__2022-01-29T00:00:00+00:00', '--job-id', '30', '--raw', '--subdir', 'DAGS_FOLDER/test.py', '--cfg-path', '/tmp/tmpqma07x4q', '--error-file', '/tmp/tmpwmknh6q6']
[2022-02-21 17:02:04,035] {standard_task_runner.py:77} INFO - Job 30: Subtask get_path
[2022-02-21 17:02:04,115] {logging_mixin.py:109} INFO - Running <TaskInstance: test_path.get_path scheduled__2022-01-29T00:00:00+00:00 [running]> on host ce809d68944a
[2022-02-21 17:02:04,197] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=test_path
AIRFLOW_CTX_TASK_ID=get_path
AIRFLOW_CTX_EXECUTION_DATE=2022-01-29T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-01-29T00:00:00+00:00
[2022-02-21 17:02:04,198] {python.py:175} INFO - Done. Returned value was: /nba_database
[2022-02-21 17:02:04,227] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=test_path, task_id=get_path, execution_date=20220129T000000, start_date=20220221T170204, end_date=20220221T170204
[2022-02-21 17:02:04,287] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-02-21 17:02:04,337] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-02-21 19:10:47,440] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: test_path.get_path scheduled__2022-01-29T00:00:00+00:00 [queued]>
[2022-02-21 19:10:47,448] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: test_path.get_path scheduled__2022-01-29T00:00:00+00:00 [queued]>
[2022-02-21 19:10:47,448] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-02-21 19:10:47,448] {taskinstance.py:1239} INFO - Starting attempt 1 of 1
[2022-02-21 19:10:47,448] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-02-21 19:10:47,459] {taskinstance.py:1259} INFO - Executing <Task(PythonOperator): get_path> on 2022-01-29 00:00:00+00:00
[2022-02-21 19:10:47,463] {standard_task_runner.py:52} INFO - Started process 114 to run task
[2022-02-21 19:10:47,466] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'test_path', 'get_path', 'scheduled__2022-01-29T00:00:00+00:00', '--job-id', '30', '--raw', '--subdir', 'DAGS_FOLDER/test.py', '--cfg-path', '/tmp/tmpgx_b3u64', '--error-file', '/tmp/tmpk_sw7x4r']
[2022-02-21 19:10:47,466] {standard_task_runner.py:77} INFO - Job 30: Subtask get_path
[2022-02-21 19:10:47,500] {logging_mixin.py:109} INFO - Running <TaskInstance: test_path.get_path scheduled__2022-01-29T00:00:00+00:00 [running]> on host 1594888d6c38
[2022-02-21 19:10:47,532] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=test_path
AIRFLOW_CTX_TASK_ID=get_path
AIRFLOW_CTX_EXECUTION_DATE=2022-01-29T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-01-29T00:00:00+00:00
[2022-02-21 19:10:47,533] {python.py:175} INFO - Done. Returned value was: ['/home/***/.local/bin', '/usr/local/lib/python37.zip', '/usr/local/lib/python3.7', '/usr/local/lib/python3.7/lib-dynload', '/home/***/.local/lib/python3.7/site-packages', '/usr/local/lib/python3.7/site-packages', '/opt/***/dags', '/opt/***/config', '/opt/***/plugins']
[2022-02-21 19:10:47,546] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=test_path, task_id=get_path, execution_date=20220129T000000, start_date=20220221T191047, end_date=20220221T191047
[2022-02-21 19:10:47,558] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-02-21 19:10:47,576] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
